src/ert/serialization/_evaluator.py

```diff
@@ -1,13 +1,15 @@
-import datetime
 import json
+from datetime import date, datetime
 from typing import Any
 
-from dateutil import parser
+from backports.datetime_fromisoformat import MonkeyPatch  # type: ignore
+
+MonkeyPatch.patch_fromisoformat()
```

Comment:
Are those the only places we do date parsing?

Reply:
No, but it was the only places that mattered according to the profiling I did